src/main/resources/xml/schema/2020-1/nordic2020-1.opf.sch

```diff
@@ -56,8 +56,8 @@
 
             <assert test="count(dc:language[not(@refines)]) = 1">[opf3c] exactly one dc:language <value-of select="if (dc:language[@refines]) then '(without a &quot;refines&quot; attribute)' else ''"
                 /> must be present in the package document.</assert>
-            <assert test="count(dc:language[not(@refines)]) = 1 and matches(dc:language[not(@refines)]/text(), '^[a-z]{2,3}(-[A-Z]{2,3})?$')">[opf3c] the language code ("<value-of
-                    select="dc:language[not(@refines)]/text()"/>") must be either a "two-letter lower case" code or a "two-letter lower case + hyphen + two- or three-letter upper case" code.</assert>
+            <assert test="count(dc:language[not(@refines)]) = 1 and matches(dc:language[not(@refines)]/text(), '^[a-z]{2,3}(-[A-Za-z0-9]+)*$')">[opf3c] the language code ("<value-of
+                    select="dc:language[not(@refines)]/text()"/>") must be either a "two-letter lower case" code or a "two-letter lower case and groups of hyphen followed by numbers or letters" (i.e. zh-Hanz-UTF8) code.</assert>
```

Comment:
three letters are still allowed though - should we still mention it in the message?

Reply:
Hi @josteinaj 

Thank you, a great catch, my bad. 

Have added a three-letter lower case as an option.

Best regards
Daniel